config/locales/javascript/javascript.en.yml

```diff
@@ -300,5 +300,8 @@ en:
       count:
         one: "1 vote so far"
         other: "<%=count%> votes so far"
+      answer_count:
+        one: "1 vote"
```

Comment:
maybe you should add zero to the english translation, even if english doesn't need it, but other languages use this key as fallback.